src/Symfony/Bundle/FrameworkBundle/Command/ContainerAwareCommand.php

```diff
@@ -53,4 +53,46 @@ public function setContainer(ContainerInterface $container = null)
     {
         $this->container = $container;
     }
+
+    /**
+     * Returns true if the service id is defined.
+     *
+     * @param string $id The service id
+     *
+     * @return bool true if the service id is defined, false otherwise
+     *
+     * @throws \LogicException
```

Comment:
What is wrong with this one ?

This exception is actually return by the `getContainer` method.